### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by hipsterusername**
@blessedcoolant Per discussion, have updated codeowners so that we're not force merging things.

This will, however, necessitate a much more disciplined approval.

### CODE REVIEW

**Review by blessedcoolant — APPROVED** [review]

**Comment by blessedcoolant** [comment]
While we still wait on original code owners for most approvals on the PR's, this will be handy in emergency cases where @hipsterusername and I have many times merged some PR's by overriding. This will just make that legal.

**Comment by Millu** [comment]
Can I be added to the README.md & mkdocs.yaml?

**Comment by hipsterusername** [comment]
> Can I be added to the README.md & mkdocs.yaml?

added to mkdocs, but no entry that would cover readme. submit a pr to that and we can approve

**Pull request merged**
